README.md

```diff
@@ -26,7 +26,7 @@ npm install --save filesniffer
 
 ## Usage
 
-Searches the cwd for files containing `myPattern`:
+Searches the current working directory for files containing the string `some string`:
```

Comment:
I've expanded `cwd` everywhere because not everyone will be familiar with the terminology.